test/CSharp7Features.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace CSharp7
+{
+  class ThrowExpressions
+  {
+    public static int ThrowExpressionExample() =>
+    3 == 3 ? 2 : throw new Exception();
```

Comment:
I'd like to keep all commits passing as I tend to prefer rebase over merge to keep history readable. You should introduce this test with the appropriate feature support.

I should document it in a contribution guide.

Reply:
What I'm used to doing for work is to let Github handle automatically squashing & rebasing every PR into a single commit. You can configure any repo to do so: https://help.github.com/en/articles/configuring-commit-squashing-for-pull-requests

That said, I can manually rebase my PRs before submitting them if that's what you prefer.